WalletWasabi.Gui/Controls/WalletExplorer/SendControlViewModel.cs

```diff
@@ -208,6 +211,8 @@ protected SendControlViewModel(WalletService walletService, string title)
 						SetAmountIfMax();
 
 						LabelToolTip = "Spending whole coins does not generate change, thus labeling is unnecessary.";
+
+						CustomChangeAddress = "";
```

Comment:
As we talked about this IRL. I am using the MAX button sometimes to check the total amount in USD then enter the specified amount. In that case, `CustomChangeAddress` is lost, moreover, because it is filled previously I might not notice it and because it is optional there won't be any warnings. 
@MaxHillebrand had some concerns about this.

Reply:
We decided on this design IRL. Please don't rehash past decisions. We could go back and forth on design decisions, but it's not productive, especially not on a hidden advanced option.